Disable IP address lookups in iptables -module when listing

mtoivo opened the pull request:
##### SUMMARY
This PR adds just `--numeric` switch to iptables when running list action (`-L`) to disable the reverse lookups of the IP addresses. 

Fixes #78793


##### ISSUE TYPE
- Feature Pull Request


##### COMPONENT NAME
iptables

##### ADDITIONAL INFORMATION
The list command in the `iptables` -module is only used internally to check the policy of a chain in `get_chain_policy()` or the existence of a chain in `check_chain_present()`. But since the list command actually also dumps the rules of a chain, iptables by default will do reverse lookups the IP addresses in the chain. And those lookups can freeze the action totally, if the defined DNS server is unreachable. 

Since in the listing the actual rules play no part in the functionality (only the policy-header is looked at or the return code of the iptables -command), the listing can be done in numeric format and it does not disturb the process.

Commit:
add --numeric to list

bcoca commented:
This is missing an option (off by default for backwards compatibility) , tests and changelog
!waiting_on_contributor

mtoivo commented:
> This is missing an option (off by default for backwards compatibility) , tests and changelog !waiting_on_contributor

So should the use of  `--numeric` switch be a parameter for the module (e.g. boolean `numeric`), or did you mean something else with the "missing an option"? That's how I first started with this, but figured it was unnecessary.

Commit:
add numeric as module parameter

Commit:
fix syntax

mtoivo commented:
**Marking as draft.**
- Added the "numeric" as module parameter (boolean) and modified to code to add `--numeric` to the listing only if the parameter `True`. Defaulting to `False`.

**TODO:**
- Add the parameter to docs
- Create changelog

Commit:
Fix indentation

Commit:
added parameter documentation

Commit:
Add parameter to iptables to disable dns lookups (#78828)

Commit:
Fix version, remove trailing whitespace
